common/realtime.py

```diff
@@ -4,16 +4,21 @@
 import time
 import multiprocessing
 
-from common.hardware import PC
+from common.hardware import PC, TICI
 from common.clock import sec_since_boot  # pylint: disable=no-name-in-module, import-error
 
 
 # time step for each process
 DT_CTRL = 0.01  # controlsd
 DT_MDL = 0.05  # model
-DT_DMON = 0.1  # driver monitoring
 DT_TRML = 0.5  # thermald and manager
 
+# driver monitoring
+if TICI:
```

Comment:
we need to handle this in service_list at some point too. our comm issue check is wrong for frontFrame/driverState/dMonitoringState for example

Reply:
Agree! Ideally these values should also be pulled from the service list.